test/utils.py

```diff
@@ -12,6 +12,13 @@
 from psslib.outputformatter import OutputFormatter
 
 
+def concat(*args):
```

Comment:
This function doesn't appear to do much, only used to save +s in call to sort? Not sure how I feel about this. Maybe wrap `sorted` as well - something like `sortedseq` that takes any number of iterables, concatenates them and calls sorted?

Reply:
I agree. `concat` doesn't bring much of a win. I will completely get rid of this function.